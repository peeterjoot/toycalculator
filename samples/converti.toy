INT32 x;
x = 1.78 + 3.86E0;
FLOAT64 f;
f = x;
PRINT f;
f = 1.78 + 3.86E0;
PRINT f;
