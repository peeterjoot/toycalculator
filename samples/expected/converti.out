4.000000
5.640000
